app/services/api/jsonrpc/jsonrpc-api.ts

```diff
@@ -20,6 +20,7 @@ export interface IJsonRpcRequest {
     fetchMutations?: boolean;
     compactMode?: boolean;
     noReturn?: boolean;
+    windowId?: string;
```

Comment:
Why are we adding this to the interface when I don't see it used anywhere?

Reply:
It uses here https://github.com/stream-labs/streamlabs-obs/pull/2694/files#diff-6287f3fd9c57c5fc50831cd0ad9ca72aR146

Reply:
Don't know how I missed that haha